recipes/dask-gateway/meta.yaml

```diff
@@ -0,0 +1,96 @@
+{% set name = "dask-gateway" %}
+{% set version = "0.2.0" %}
+
+package:
+  version: {{ version }}
+  name: {{ name|lower }}
+
+source:
+  url: https://github.com/jcrist/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 1ae59cc611fb4a7dd1210bdf82e1eef151d450f144ae66d1d478582cd9f95964
+
+build:
+  number: 0
+  version: {{ version }}
+
+outputs:
+  - name: dask-gateway-base
+    script: dask-gateway-client.sh
+    build:
+      entry_points:
+        - dask-gateway-scheduler = dask_gateway.dask_cli:scheduler
+        - dask-gateway-worker = dask_gateway.dask_cli:worker
+    requirements:
+      host:
+        - python
+        - pip
+      run:
+        - python
+        - tornado
+        - dask >=2.0.0
+        - distributed >=2.0.0
+    test:
+      imports:
+        - dask_gateway
+      commands:
+        - dask-gateway-scheduler --help
+        - dask-gateway-worker --help
+  - name: dask-gateway-kerberos
+    requirements:
+      - {{ pin_subpackage('dask-gateway-base', exact=True) }}
+      - pykerberos
+
+  - name: dask-gateway-server-base
+    script: dask-gateway-server.sh
+    build:
+      entry_points:
+        - dask-gateway-server = dask_gateway_server.app:main
+        - dask-gateway-jobqueue-launcher = dask_gateway_server.managers.jobqueue.launcher:main
+    requirements:
+      host:
+        - python
+        - go
```

Comment:
Does this include compiled go code?

Reply:
Yes.

Reply:
it is compiling go code

Reply:
https://github.com/jcrist/dask-gateway/blob/0.2.0/dask-gateway-server/setup.py#L36

Reply:
Then ignore my `python >=3` suggestions and use `skip: true  # [py2k]` in the `build:` section